### PR TITLE
Documentation ashukla

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,52 +7,50 @@
 
 ## Overview
 
-### Introduction
-
-*MITRE ATT&CK Threat Hunting Solution Pack* is designed to provide a set of threat hunting playbooks that demonstrates to users a variety of scenarios and use cases around threat hunting based on the information provided by the MITRE ATT&CK Framework.
+*MITRE ATT&CK Threat Hunting Solution Pack* is designed to provide a set of threat hunting playbooks that demonstrate a variety of scenarios and use cases around threat hunting based on the information provided by the MITRE ATT&CK Framework.
```
Comment (cs-suyog-jinturkar) [inline README.md]: Introduction heading is removed. We will need it same as SoC sim

File: README.md [hunk 2]
```diff
@@ -9,52 +7,50 @@
...
 Configure the data ingestion from SIEM using connectors such as Elasticsearch or Splunk. The hunting playbooks then trigger the hunt for the specified period mentioned in the hunt record and create an alert for the corresponding MITRE Technique.
 
-Refer to the following Simulation Scenarios' to experience the use case without any SIEM configuration.
+Refer to the following Simulation Scenarios to experience the use case without any SIEM configuration.
 
-### Usage
+## Usage
 
-This Solution Pack ships with the following simulation scenarios. [Refer](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md) to Simulate Scenario documentation to understand how to Simulate and Reset Scenario.
+This Solution Pack ships with the following simulation scenarios. Refer to [Simulate Scenario documentation](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md) to understand how to Simulate and Reset scenarios.
```
Comment (cs-suyog-jinturkar) [inline README.md]: This Solution Pack ships with the following simulation scenarios. --> this stmt will not be present here... It should be after this para.

File: README.md [hunk 3]
```diff
@@ -9,52 +7,50 @@
...
 
-### Introduction
-
-*MITRE ATT&CK Threat Hunting Solution Pack* is designed to provide a set of threat hunting playbooks that demonstrates to users a variety of scenarios and use cases around threat hunting based on the information provided by the MITRE ATT&CK Framework.
+*MITRE ATT&CK Threat Hunting Solution Pack* is designed to provide a set of threat hunting playbooks that demonstrate a variety of scenarios and use cases around threat hunting based on the information provided by the MITRE ATT&CK Framework.
 
 Configure the data ingestion from SIEM using connectors such as Elasticsearch or Splunk. The hunting playbooks then trigger the hunt for the specified period mentioned in the hunt record and create an alert for the corresponding MITRE Technique.
 
-Refer to the following Simulation Scenarios' to experience the use case without any SIEM configuration.
+Refer to the following Simulation Scenarios to experience the use case without any SIEM configuration.
```
Comment (cs-suyog-jinturkar) [inline README.md]: This line and "This Solution Pack ships with the following simulation scenarios." are saying same thing again and again. Can we combine it?

File: README.md [hunk 4]
```diff
@@ -9,52 +7,50 @@
...
 
-### Usage
+## Usage
 
-This Solution Pack ships with the following simulation scenarios. [Refer](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md) to Simulate Scenario documentation to understand how to Simulate and Reset Scenario.
+This Solution Pack ships with the following simulation scenarios. Refer to [Simulate Scenario documentation](https://github.com/fortinet-fortisoar/solution-pack-soc-simulator/blob/develop/docs/solution-pack-guide.md) to understand how to Simulate and Reset scenarios.
```
Comment (cs-suyog-jinturkar) [inline README.md:21]: We can add "This Solution Pack ships with the following simulation scenarios." here just before the scenrion details starts...

File: README.md [hunk 5]
```diff
@@ -9,52 +7,50 @@
...
 
-The scenario demonstrates threat hunting for the attacks using the MITRE technique Signed Binary Proxy Execution [T1218] and generates demo Hunt, Alert and Technique/Sub-Technique records
+The scenario demonstrates threat hunting for the attacks using the MITRE technique Signed Binary Proxy Execution [T1218] and generates demo Hunt, Alert, and Technique/Sub-Technique records
 
-Goto generated alerts and observe the following:
+Navigate to generated alerts and observe the following:
 
-- MITRE Technique information (MITRE ID, Technique and Sub-Technique etc) is presented as a handy reference
-- Attack details (Process ID, Process Name, File Path, Computer Name etc.) presented for analyzing the case
+- MITRE Technique information (MITRE ID, Technique, or Sub-Technique) is presented as a handy reference
```
Comment (cs-suyog-jinturkar) [inline README.md]: and is correct here..

File: README.md [hunk 6]
```diff
@@ -9,52 +7,50 @@
...
+- Attack details (Process ID, Process Name, File Path, or Computer Name) presented for analyzing the case
```
Comment (cs-suyog-jinturkar) [inline README.md]: and is needed instead of or

File: docs/content.md [hunk 7]
```diff
@@ -0,0 +1,43 @@
+# MITRE ATT&CK Threat Hunting
+
+# Overview
+
+The MITRE ATT&CK Threat Hunting Solution Pack demonstrates a variety of scenarios and use cases around threat hunting using the information provided by the MITRE ATT&CK Framework. 
+
+*MITRE ATT&CK is a globally-accessible knowledge base of adversary tactics and techniques based on real-world observations. The ATT&CK knowledge base issued as a foundation for the development of specific threat models and methodologies in the private sector, in government, and in the cybersecurity product and service community.*
```
Comment (cs-suyog-jinturkar) [inline docs/content.md]: why new file? can we add this in same readme?

File: docs/content.md [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+
+### Solution Packs
+
+This Solution Pack requires the deployment of the following solution packs.
+
+<table>
+    <thead>
+        <tr>
+            <th>Solution Pack</th>
+            <th>Purpose</th>
```
Comment (cs-suyog-jinturkar) [inline docs/content.md]: This is diff formatting other than pipes which we used else where... what is a reason for not using pipe here?

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: docs/content.md (hunk 7 above)
Comment (cs-abhishek-shukla) [reply docs/content.md]: Deleted

File: docs/content.md (hunk 8 above)
Comment (cs-abhishek-shukla) [reply docs/content.md]: removed

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**